test/commit-message-config.integration-test.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
```

Comment:
I like the rename of git -> commit-message-config, and I like the separation into integration tests, and regular tests too.